lighthouse-cli/test/smokehouse/test-definitions/pwa/pwa2-expectations.js

```diff
@@ -46,9 +46,8 @@ module.exports = [
         // Ignore speed test; just verify that it ran.
         },
         'installable-manifest': {
-          score: 0,
-          details: {items: [jakeExpectations]},
-          explanation: /^Failures: .*short_name/,
+          score: 1,
```

Comment:
does this mean "`Manifest does not contain a 'name' or 'short_name' field`" only requires one of `name` and `short_name`, so is passing now?

Reply:
😆 

a coupla weeks ago i spent a while looking at https://source.chromium.org/chromium/chromium/src/+/master:chrome/browser/installable/installable_manager.cc;l=651-655;drc=05dff50d9ce188e04ef796f0f1c28fbef09ace59 and my brain still doesn't make sense of it.

it seeems to say:

```js
if (nameIsMissingOrEmpty && shortNameIsMissingOrEmpty) {
  errors.push('MANIFEST_MISSING_NAME_OR_SHORT_NAME')
}
```

and how does && in the logic match up with a var named _OR_.


in fact it seemed so wrong that i figured it might be a bug. so adriana and I made a CL: https://chromium-review.googlesource.com/c/chromium/src/+/2530481  
but the tests are all failureful.

so tbh i have no idea.

Reply:
The fact that this makes sense to me now has me worried about my understanding... 😆 

I think the `MANIFEST_MISSING_NAME_OR_SHORT_NAME` is just ambiguously named because when interpreting it as "manifest missing something - name or short name needed", which is consistent with my understanding that you need *something* to use as the name, but you definitely don't need both name and short_name.

It's not the case for installability that you need a `short_name` so I would expect the `&&` to fail the tests.